src/tovnode.ts

```diff
@@ -24,7 +25,16 @@ export function toVNode(node: Node, domApi?: DOMAPI): VNode {
     for (i = 0, n = elmChildren.length; i < n; i++) {
       children.push(toVNode(elmChildren[i], domApi));
     }
-    return vnode(sel, { attrs }, children, undefined, node);
+    const data = { attrs };
+    if (
+      sel[0] === "s" &&
+      sel[1] === "v" &&
+      sel[2] === "g" &&
+      (sel.length === 3 || sel[3] === "." || sel[3] === "#")
```

Comment:
```suggestion
      sel === "svg" ||
      sel.startsWith("svg.") ||
      sel.startsWith("svg#")
```
A bit easier to read

Reply:
Do you want me to change that here too? https://github.com/snabbdom/snabbdom/blob/9de2307dd9bec6106f587a18eb7b0c994e24722b/src/h.ts#L79-L84

Reply:
yes, please

Reply:
Cool, do you want me to create a new commit or to squash?

Reply:
Thinking a bit more about this, this version is a bit more performant. I applied my other suggestion and will merge it directly now